src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -36,6 +37,7 @@ internal BindingTemplate(string pattern, IReadOnlyList<BindingTemplateToken> tok
             _pattern = pattern;
             _tokens = tokens;
             _ignoreCase = ignoreCase;
+            _parameterResolver = parameterResolver ?? new DefaultParameterResolver();
```

Comment:
This logic isn't pluggable today, so I don't think we need to worry about this now. I've taken it 90% of the way. I'm confident we can do this as an add on later.

However, there might still be a way for me to get this in place now, if we made this available statically (e.g. similar to System.Web.Http.GlobalConfiguration). Something like that would allow me to access from BindingTemplate internally and get this working for all bindings.

I'll take this up in a separate PR/work item, since this is an additive feature.